### PR TITLE
Add a new question to the "Study information" page

### LINKED ISSUE #4467: Add a new question to the "Study information" page
Following a request from @LiamHart-hub, the "Study information" page of the applications app should include a new question. The user should select either one of two answers to the question. The question and the answers should be located beneath the text added by #4466.

The question is as follows:

> Have all applicants for OpenSAFELY read and agree to [OpenSAFELY's Analytic Methods Policy](https://www.opensafely.org/policies-for-researchers/#opensafely-analytic-methods-policy)?

(The link to OpenSAFELY's Analytic Methods Policy is important!)

The answers are as follows:

> * Yes
> * No

The question is analogous to the "Have all applicants for OpenSAFELY access completed the Getting Started tutorial?" question on the "Software development experience" page.

https://github.com/opensafely-core/job-server/blob/f4e23fbd99a3482ebb8dfecbb97c6b43ac188ba8/applications/form_specs.py#L478-L487

### PR DESCRIPTION
**Pull request by iaindillingham**
Closes #4467

Here's the question:

Analytic Methods Policy

Notice that the link is in the help text below the question, not in the question itself. `Field.label` isn't parsed, but `Field.help_text` is. I've asked Liam whether this suffices.[^1]

And here's the question in the context of the page:

Study information

No extra work is required to show the answer to the question in the staff area. The default answer is `null`.

[^1]: https://bennettoxford.slack.com/archives/C068NDYALSF/p1721748150730279

### CODE REVIEW

File: applications/forms.py [hunk 1]
```diff
@@ -12,6 +12,7 @@ def __init__(self, *args, **kwargs):
         # which uses choices.
         radio_fields = [
             "all_applicants_completed_getting_started",
+            "read_analytic_methods_policy",
```
Comment (iaindillingham) [inline applications/forms.py:15]: This! 😱

Comment (Jongmassey) [reply applications/forms.py:15]: IKR, took me a second to work out why this was needed

Comment (iaindillingham) [reply applications/forms.py:15]: It made me very sad.

File: applications/form_specs.py [hunk 2]
```diff
@@ -113,6 +113,11 @@
                         help_text=snippet("study_purpose-help_text"),
                         template_name="form_textarea",
                     ),
+                    Field(
+                        name="read_analytic_methods_policy",
+                        label="Have all applicants for OpenSAFELY read and agreed to the OpenSAFELY Analytic Methods Policy?",
```
Comment (Jongmassey) [inline applications/form_specs.py:118]:
```suggestion
                        label="Have all applicants read and agreed to the OpenSAFELY Analytic Methods Policy?",
```

I think the "for OpenSAFELY" feels implicit here, but I'm aware there's been a bit of wrangling over the wording already so happy for you to disregard!

**Review by Jongmassey — APPROVED** [review]
Technically all is good and save for a minor opinion I think the wording is also fine

File: applications/form_specs.py (hunk 2 above)
Comment (iaindillingham) [reply applications/form_specs.py:118]: Sadly, this isn't negotiable. I share your snag, however.

**Pull request merged**